### ISSUE TITLE
space is not working for searches

### ISSUE THREAD

**Issue by Bram-diederik** [ondras/cyp#31]
I can find the chemical brothers when i type "the " if i enter "the c" no artists are found.

**Comment by ondras** [comment]
According to the source code (https://github.com/ondras/cyp/blob/a6df882dcb511be1ef83d16c4a35f7b0dd2b06f0/app/js/elements/library.ts#L183, https://github.com/ondras/cyp/blob/a6df882dcb511be1ef83d16c4a35f7b0dd2b06f0/app/js/mpd.ts#L111), the input is passed to MPD verbatim. I would say that you should ask the MPD dev team regarding the `contains` operator (https://mpd.readthedocs.io/en/latest/protocol.html#filters).
In other words, there is very little we can do cyp-wise.

**Comment by Bram-diederik** [comment]
I retested it now it works very strange.
